src/api/app/assets/stylesheets/webui/application/bento/base.scss

```diff
@@ -177,6 +177,21 @@ blockquote{border-left:5px solid #690;color:#666;font-style:italic;}
         }
     }
 }
+#login-form {
```

Comment:
why not adding this in https://github.com/openSUSE/open-build-service/pull/4661/files#diff-683e130277e4182c9a0b2f8636c08b1aR138 ? :thinking: